data/Zieminska_2015/metadata.yml

```diff
@@ -125,7 +125,7 @@ config:
     taxon_name: species
     site_name: site
     date: sampling date
-  custom_R_code: data %>% mutate(vessel_diameter = 2*(sqrt(`vessel area (micron^2)`/3.14159)))
+  custom_R_code: data %>% mutate(vessel_diameter = 2*(sqrt(`vessel area (micron^2)`/3.14159))) %>% mutate(`sampling date` = `sampling date` %>% ym() %>% format.Date("%Y-%m"))
```

Comment:
`format.Date("%Y-%m")` is included for all the studies where the resolution is at the `month` not the `day`. Otherwise, even if the data are input as 2020-April, they are output as 2020-04-01. Since the output is in character format, this is a way to change the resolution back to `month`. This is why I was contemplating if it wasn't better to have separate fields for `year`, `month`, and `day`, so you only fill in the relevant ones, but this scheme works fine for now. (You'll notice that Geange_2020, two comments above, has the same call to format.)

Reply:
Great, sounds like a good approach